spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -214,4 +214,24 @@
       end
     end
   end
+
+  context 'when `AllowedPatterns:` and `Prefixes:` are both empty' do
+    let(:cop_config) do
+      { 'Prefixes' => [], 'AllowedPatterns' => [] }
+    end
+
+    it 'skips any description' do
+      expect_no_offenses(<<~RUBY)
+        context 'this is valid patterns context' do
```

Comment:
```suggestion
        context 'arbitrary text' do
```
Nitpick: there are no patterns to validate against